command_helper/noralize_cell.py

```diff
@@ -0,0 +1,93 @@
+import re
+try:

```

Comment:
The same as above and in other modules: no need to have imports for both sublime2 and sublime3